uiServices/src/testCaseServices.go

```diff
@@ -150,19 +151,22 @@ func getAllTestCases(rw http.ResponseWriter, req *http.Request) {
 
 			file, err := os.Open(fmt.Sprintf("%s%s", testCasePathDir, filename))
 			if err != nil {
-				logrus.Error("Cannot Open File: " + filename)
+				logrus.Error("File not found: " + filename)
```

Comment:
Would it not be better to say something like:
`logrus.Error("Error opening the file: ",  err )`
I would also add the error as in example.